content/blog/accessing-dtap-in-pods.md

```diff
@@ -0,0 +1,211 @@
+
+In this blog, I will introduce two ways to access DataTaps in Kubernetes. The first method will be accessing the DataTaps using HDFS Commands and the second method will be directly reading data from Apache Spark (using pyspark). Here we go.
+
+## **Enable DataTap when creating the pod**
+First and foremost, we have to enable DataTaps of the application. This can be done by ticking the "Enable DataTap" box when creating the application.
+
+![image](https://user-images.githubusercontent.com/72959956/119443704-9cc92180-bd5c-11eb-8fce-b6b53823336c.png)
+
+This will result in mounting a lot of files at ```/opt/bdfs/``` of your pod. If you can see the files, shown below, in your pod, this means that your pod is DataTap enabled, and you are now ready to access the files in DataTap.
+
+![image](https://user-images.githubusercontent.com/72959956/120776952-58593500-c557-11eb-9dcd-4146d581a761.png)
+
+
+The generic approach can be concluded into these two steps:
+1. Add ```/opt/bdfs/bluedata-dtap.jar``` to the classpath.
+2. Configure Hadoop with the following values.
+
+| name | value |
+|---|---|
+| fs.dtap.impl | com.bluedata.hadoop.bdfs.Bdfs |
+| fs.AbstractFileSystem.dtap.impl | com.bluedata.hadoop.bdfs.BdAbstractFS |
+| fs.dtap.impl.disable.cache* | false |
+
+*optional
+
+> Reference:
+> - [Accessing DataTaps in Kubernetes Pods](https://docs.containerplatform.hpe.com/53/reference/kubernetes/tenant-project-administration/datataps/Accessing_DataTaps_in_Kubernetes_Pods.html)
+
+## Uniform Resource Identifier
+In HPE Ezmeral Container Platform, you can see different types of file system used by the shared storage resources. You can manage different data source through a GUI and representing files with same URI. The URI will be in the format of 
+```
+dtap://datatap_name/some_subdirectory/another_subdirectory/some_file
+```
+
+|Screenshot|Description|
+|---|---|
+|![image](https://user-images.githubusercontent.com/72959956/121467168-35150680-c9eb-11eb-901c-77e83097cdf9.png)| You can manage different data source whether they are in MapR filesystem or HDFS|
+|![image](https://user-images.githubusercontent.com/72959956/121467262-5f66c400-c9eb-11eb-958d-911f18281a27.png)| You can upload, delete or rename files using GUI|
+
+# Access DataTaps using HDFS commands
+## Introduction
+The Hadoop distributed file system (HDFS) is the key component of the Hadoop ecosystem. HDFS commands, of course, are the commands which responsible for manipulate files for HDFS. 
+
+To use the HDFS commands, first you need to start the Hadoop services using the following command:
+
+## Prepare Hadoop
+Some of the Kubedirector App provided by HPE is pre-installed a well-configured Hadoop for you. Hence, the following installation steps can be skipped.
+### Install OpenJDK and the dependency
+```bash
+apt update && apt upgrade -y
+apt install wget -y
+
+# install openjdk
+DEBIAN_FRONTEND=noninteractive apt-get install openjdk-11-jdk-headless -y
+```
```

Comment:
make sure to insert a line break after the code block and before the next header level 3.